examples/03_connectivity/plot_fast_clustering.py

```diff
@@ -0,0 +1,149 @@
+"""
```

Comment:
Can you remove this example and merge it into examples/03_connectivity/plot_rest_parcellations.py, as a new section at the end, modeled on the previous sections.

Reply:
@ja-che ?

Reply:
That is what I tried to do, I just forget to remove plot_fast_clustering.py